src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,41 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  height: 100%;
+  min-width: $iui-3xl * 2;
+  width: $iui-3xl * 4;
+  max-width: 50vw;
+  box-sizing: border-box;
+  padding: $iui-baseline $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
+  resize: horizontal;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: max-width $iui-speed-fast ease-out;
+  }
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  .iui-side-navigation-title {
+    @include iui-text(title);
+    margin-bottom: $iui-baseline;
+    user-select: none;
+  }
+}
+
+@mixin iui-side-navigation-wrapper {
+  display: flex;
+  position: relative;
```

Comment:
Not needed

Reply:
I guess we can have it.. if any absolute elements appear in children, they could be relative to the side nav...?